.github/workflows/build-test-deploy.yml

```diff
@@ -1237,12 +1222,12 @@ jobs:
             nix run "github:ComposableFi/composable/${{ github.event.pull_request.head.sha }}#composable-node" -L
             ```
 
-            #### Spin up a local devnet (Dali + Rococo):
+            #### Spin up a local devnet (Rococo + Dali + Karura + Statemine):
```

Comment:
that may be named just Release, because later there will be Hyperspace, and Picasso running at same time.